content/en/real_user_monitoring/funnel_analysis/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Funnel Analysis
+kind: documentation
+disable_toc: false
+further_reading:
+- link: "/real_user_monitoring/explorer/"
+  tag: "Documentation"
+  text: "RUM Explorer"
+---
+
+## Overview
+
+Funnel analysis helps you track conversion rates across key workflows to identify and address any bottlenecks. Specifically, you can:
+
+- See if customers drop off at a certain point due to poor website performance
+- Track how conversion rate changes over time as new features are built
+- Measure the impact of how adding new steps to a workflow impacts dropoff rate
+
+## Building a funnel
+
+To build a funnel, select your starting view or action and click on the plus icon to proceed in building additional steps. You can also use drag and drop functionality to move steps around. 
+
+{{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-building-a-funnel-1.mp4" alt="Filtering network map with search" video=true >}}
+
```

Comment:
@rtrieu  Forgot to add this sooner, but perhaps here we add a comment about what conversion rate means. Suggesting something along the lines of this:
_Any action or views can happen between two steps in a funnel and won't impact the step-by-step, nor overall conversion rate. As long as step1 and step2 happen in the right order in a given session at least once, this will count as a single converted session_

Reply:
Hi @jamiemilstein, this sentence confuses me a bit. Can you please clarify what is meant by "the right order"?

So far, I've added this sentence to explain conversion rates in the overview:

**Note:** The conversion rate is the number of visitors to your website that completed a desired goal (a conversion) out of the total number of visitors.